src/components/NoFieldSetOrLegendForm.tsx

```diff
@@ -0,0 +1,76 @@
+import { useRouter } from "next/router";
+import React, { FunctionComponent, ReactNode } from "react";
+import { FormError } from "../lib/form/FormManager";
+import { BackButton, Button } from "./Button";
+import { FormErrorSummary } from "./ErrorSummary";
+import { Form, FormGroup } from "./Form";
+import { Grid } from "./Grid";
+import { Layout } from "./Layout";
+import { IfYouNeedHelp } from "./Mca";
+import { GovUKBody } from "./Typography";
+
+interface NoFieldSetOrLegendFormProps {
+  children: ReactNode;
+  previousPageUrl: string;
+  pageHeading: string;
+  showCookieBanner: boolean;
+  formErrors?: FormError[];
+  errorMessages?: string[];
+  pageText?: string | ReactNode;
+  includeUseIndex?: boolean;
+}
+
+export const NoFieldSetOrLegendForm: FunctionComponent<NoFieldSetOrLegendFormProps> =
+  ({
+    children,
+    previousPageUrl,
+    pageHeading,
+    showCookieBanner,
+    formErrors = [],
+    errorMessages = [],
+    pageText = null,
+  }: NoFieldSetOrLegendFormProps): JSX.Element => {
+    const pageTextComponent: ReactNode =
+      typeof pageText === "string" ? (
+        <GovUKBody>{pageText}</GovUKBody>
+      ) : (
+        pageText
+      );
+
+    return (
+      <Layout
+        navigation={<BackButton href={previousPageUrl} />}
+        title={pageHeading}
+        showCookieBanner={showCookieBanner}
+      >
+        <Grid
+          mainContent={
+            <>
+              <FormErrorSummary formErrors={formErrors} />
+              <Form>
+                <FormGroup errorMessages={errorMessages}>
+                  <h1 className="govuk-heading-l govuk-!-margin-bottom-3">
```

Comment:
Can tidy this up to use a `const headerComponent` at the top of the form and render that here in the template

Reply:
`NoFieldsetOrLegendForm` component removed